src/sections/image_resources_section.rs

```diff
@@ -113,24 +107,35 @@ impl ImageResourcesSection {
 
         let version = cursor.read_i32()?;
         if version != 6 {
-            unimplemented!("Adobe Photoshop 6.0+ slice currently unsupported");
+            unimplemented!(
+                "Only the Adobe Photoshop 6.0 slices resource format is currently supported"
+            );
         }
 
-        // We do not currently parse top of all the slices, skip it
-        cursor.read_i32()?;
-        // We do not currently parse left of all the slices, skip it
-        cursor.read_i32()?;
-        // We do not currently parse bottom of all the slices, skip it
-        cursor.read_i32()?;
-        // We do not currently parse right of all the slices, skip it
-        cursor.read_i32()?;
-        // We do not currently parse name of group of slices, skip it
-        cursor.read_unicode_string()?;
+        let _top = cursor.read_i32()?;
+        let _left = cursor.read_i32()?;
+        let _bottom = cursor.read_i32()?;
+        let _right = cursor.read_i32()?;
+
+        let _group_of_slices_name = cursor.read_unicode_string()?;
+
+        let mut number_of_slices = cursor.read_u32()?;
+
+        // For some reason one of our test files that has no slices is returning that it has
+        // [0, 1, 0, 0] as the number of slices (65536).
+        // We're asserting that we parse the correct number of bytes for the section, so it
+        // doesn't _seem_ to be an issue with reading the wrong data.
+        //
+        // Would be great to find a better solution for this
+        //
+        // @see tests/slices_resource.rs
+        if number_of_slices == 65536 {
```

Comment:
I just have no idea .. But once I add this both the test file as well as the file in my real project work.

My only guess is that `[0, 1, 0, 0]` for the number of slices means something.